### DISCUSSION TITLE
Using Lemminx extensions

### DISCUSSION THREAD

**Discussion by dyzcypul** [redhat-developer/vscode-xml#911] [Q&A]
Is there any way to see that the extension was loaded?  I installed vscode-xml and vscode-xml-maven

running JSP.exe I can see 
22548 org.eclipse.lemminx.XMLServerLauncher -DwatchParentProcess=false -XX:+ExitOnOutOfMemoryError -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=c:\Users\devxw\AppData\Roaming\Code - Insiders\User\globalStorage\redhat.vscode-xml -Xmx64M

When opening an empty pom.xml, I get no extra completion items than what is default.  I would have expected to see the completion item "minimal pom content" from createMinimalPOMCompletionSnippet in lemminx-maven extension code.

**Comment by angelozerr** [comment]
[vscode-xml-maven](https://github.com/angelozerr/vscode-xml-maven) is just a POC and it hosts an old version of lemminx-extension.

Today if you wish to test lemminx maven, you need to download the lemminx maven uber JAR. I create an issue for that at https://github.com/eclipse/lemminx-maven/issues/430

But we hope that vscode-maven will embed lemminx maven, see https://github.com/microsoft/vscode-maven/issues/47
